.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -76,4 +76,4 @@ Base links to copy
 
 ### Link to Definiton of Done
 
-More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)
+More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.dbildungscloud.de/x/mAITD)
```

Comment:
It is better to remove this for an Uniform template designs with hpi-schul-cloud/nuxt-client ?